packages/react-native-windows-init/src/Cli.ts

```diff
@@ -337,88 +357,90 @@ function isProjectUsingYarn(cwd: string): boolean {
       process.exit(EXITCODE_INCOMPATIBLE_OPTIONS);
     }
 
-    if (!version) {
-      const rnVersion = getReactNativeVersion();
-      version = getDefaultReactNativeWindowsSemVerForReactNativeVersion(
-        rnVersion,
-      );
-    }
-
-    const rnwResolvedVersion = await getLatestMatchingRNWVersion(version);
-
-    if (!rnwResolvedVersion) {
-      if (argv.version) {
-        console.warn(
-          `Warning: Querying npm to find react-native-windows@${
-            argv.version
-          } failed.  Attempting to continue anyway...`,
-        );
-      } else {
-        const rnwLatestVersion = await getLatestRNWVersion();
-        console.error(
-          `
-No compatible version of ${chalk.green('react-native-windows')} found.
-The latest supported version is ${chalk.green(
-            'react-native-windows',
-          )}@${chalk.cyan(rnwLatestVersion)}.
-Please modify your application to use ${chalk.green(
-            'react-native',
-          )}@${chalk.cyan(
-            getMatchingReactNativeSemVerForReactNativeWindowsVersion(
-              rnwLatestVersion,
-            ),
-          )} or another supported version of ${chalk.green(
-            'react-native',
-          )} and try again.
-`,
+    if (!useDevMode) {
+      if (!version) {
+        const rnVersion = getReactNativeVersion();
+        version = getDefaultReactNativeWindowsSemVerForReactNativeVersion(
+          rnVersion,
         );
-        process.exit(EXITCODE_NO_AUTO_MATCHING_RNW);
       }
-    }
 
-    if (!argv.version) {
-      console.log(
-        `Latest matching version of ${chalk.bold(
-          'react-native-windows',
-        )} for ${chalk.green('react-native')}@${chalk.cyan(
-          getReactNativeVersion(),
-        )} is ${chalk.green('react-native-windows')}@${chalk.cyan(
-          rnwResolvedVersion,
-        )}`,
-      );
+      const rnwResolvedVersion = await getLatestMatchingRNWVersion(version);
 
-      if (rnwResolvedVersion && semver.prerelease(rnwResolvedVersion)) {
-        const rnwLatestVersion = await getLatestRNWVersion();
-        console.warn(
-          `
-${chalk.green('react-native-windows')}@${chalk.cyan(
-            rnwResolvedVersion,
-          )} is a ${chalk.yellow('pre-release')} version.
-The latest supported version is ${chalk.green(
+      if (!rnwResolvedVersion) {
+        if (argv.version) {
+          console.warn(
+            `Warning: Querying npm to find react-native-windows@${
+              argv.version
+            } failed.  Attempting to continue anyway...`,
+          );
```

Comment:
newWarn?

Reply:
That's only in the CLI

Reply:
Similar as above, utils are not available in this part of the CLI.